BraintreeUIKit/Models/BTUIKCardType.m

```diff
@@ -134,37 +111,37 @@ + (NSArray *)allCards
 
         BTUIKCardType *visa = [[BTUIKCardType alloc] initWithBrand:BTUIKLocalizedString(CARD_TYPE_VISA)
                                                   securityCodeName:BTUIKLocalizedString(CVV_FIELD_PLACEHOLDER)
-                                                          prefixes:@[@"4"]];
+                                                          prefixes:@[@"^4\\d*"]];
```

Comment:
I also debated this. I kinda like the idea of having multiple regex patters (as a possibility). If we ever needed it, it could aid with the readability of the regex patterns. But in either case, `prefixes` isn't quite right. But it isn't _totally_ incorrect since the regex primary search on bin ranges (prefixes).

> The leading six digits of the card number is the "issuer identification number (IIN)", sometimes referred to as the "bank identification number (BIN)".